map.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <cstdint>
+#include "raycaster_data.h"
```

Comment:
The header `raycaster_data.h` is implementation-specific. You should not include in this header. Instead, leave it to C++ files.